Leftpad.rb

```diff
@@ -1,7 +1,33 @@
-stringIn = gets.chomp
-timesIn = gets.to_i
-charIn = gets.chomp
+def left_pad(string = "You are extremely lazy, Kid", times = 1, char = " ")
+  puts "Choose a word:"
+  string_in = gets.chomp
+  puts "Choose a number:"
+  times_in = gets.to_i
+  puts "Choose a character:"
+  char_in = gets.chomp
+  puts "Would you like to have your word of choice before all the other characters? Y/N"
+  bool = gets.chomp
+  bool.downcase!
 
-chars = charIn * timesIn
+  if string_in == nil || string_in == ""
```

Comment:
is there any case where string_in will be in fact nil?

Reply:
I can see now how I had this very unreasonable idea of fitting my JS logic 1:1 into Ruby project (especially since `gets` and `prompt()` are not equivalent); I'll correct that.